src/createCollapsibleTabs.tsx

```diff
@@ -130,6 +131,8 @@ const createCollapsibleTabs = <T extends ParamList>() => {
       const isGliding = useSharedValue(false)
       const endDrag = useSharedValue(0)
       const calculateNextOffset = useSharedValue(index.value)
+      const headerScrollDistance =
```

Comment:
This will be recomputed on every rerender, shouldn't we use `useSharedValue` or `useDerivedValue` or `useState`? What you think?

Reply:
Maybe useDerivedValue is the way to go.

Reply:
Yeah, makes complete sense, my mistake, fixed